.changes/next-release/Bug Fix-c76a701b-a106-46ef-99d9-5bb0760c469a.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "FeatureDev: Rejected files are excluded when generated code is accepted by the customer"
```

Comment:
does "FeatureDev" mean anything to customers? In the future I would word it as "Amazon Q FeatureDev", or something like that.

Reply:
got it thanks